Modules/Sources/WordPressUI/Components/RestApiUpgradePrompt.swift

```diff
@@ -0,0 +1,63 @@
+import SwiftUI
+import DesignSystem
+
+public struct RestApiUpgradePrompt: View {
+
+    private let localizedFeatureName: String
+    private var didTapGetStarted: () -> Void
+
+    public init(localizedFeatureName: String, didTapGetStarted: @escaping () -> Void) {
+        self.localizedFeatureName = localizedFeatureName
+        self.didTapGetStarted = didTapGetStarted
+    }
+
+    public var body: some View {
+        VStack {
+            let scrollView = ScrollView {
+                VStack(alignment: .leading) {
+                    Text(Strings.title)
+                        .font(.largeTitle)
+                        .fontWeight(.semibold)
+                        .padding(.bottom)
+
+                    Text(Strings.description(localizedFeatureName: localizedFeatureName))
+                        .font(.body)
+                }.padding()
+            }
+
+            if #available(iOS 16.4, *) {
+                scrollView.scrollBounceBehavior(.basedOnSize, axes: [.vertical])
+            }
```

Comment:
@jkmassel I didn't use [the `AccessibilityScrollView` in your PR](https://github.com/wordpress-mobile/WordPress-iOS/blob/add/users-for-self-hosted-sites/Modules/Sources/WordPressUI/Components/AccessibilityScrollView.swift#L8-L9), because dynamic size seems working fine without it. Let me know if I missed anything.

Reply:
Not a problem – we have a lot more room to work with here